include/cantera/kinetics/InterfaceKinetics.h

```diff
@@ -19,6 +19,7 @@ namespace Cantera
 class SurfPhase;
 class ImplicitSurfChem;
 class InterfaceReaction;
+class BMInterfaceReaction;
```

Comment:
Does `Interface` need to be there? Similar to your comment about the input file name, if `BlowersMaselReaction` would be specific enough that seems good to me.

As an aside, I can't help but think of the potty humor meaning of BM.

Reply:
Yes, it does, because this PR also adds the `BlowersMaselReaction` type for gas phase reactions, which is the same thing but without the support for coverage dependencies.

Reply:
In that case, I don't think it's all that long winded to include the `Interface` in there. This isn't something that people are going to type all that often and modern IDEs support code completion anyways.